cmake/modules/ECMAddAppIcon.cmake

```diff
@@ -1,106 +1,89 @@
-#.rst:
-# ECMAddAppIcon
```

Comment:
Why all these changes? The seem to be not needed to fix the path issue?

Reply:
Okay, I see you updated the modules.

Reply:
Maybe in the future instead of copying this stuff around, we can make ECM a git submodule. That will also prevent mistakes like the one below I mentioned.